test/tests/functional/pbs_ralter.py

```diff
@@ -317,8 +325,13 @@ def alter_a_reservation(self, r, start, end, shift=0,
                         Default: 1
         :type  sequence: int.
 
-        :param a_duration: The duration to modify
-        :type a_duration: int
+        :param a_duration: The duration to modify.
+        :type a_duration: int.
+        :param force: Force ralter.
```

Comment:
docstring says force but the function declaration says extend

Reply:
Good catch!